### ISSUE TITLE
DBUS no reply

### ISSUE THREAD

**Issue by Vic3198** [AsamK/signal-cli#137]
Hi,
I have a problem trying to run signal-cli via system bus.

I have set up the daemon, with signal-cli user and all, but when I try to run, as root user, or signal-cli, or pi (yes I'm on a Raspberry Pi), the following :
`signal-cli --dbus-system send -m 'test' +33XXXXXXXXX`
I get
`Cannot connect to dbus: No reply within specified time`
`Aborting.`

Both `signal.service` and `dbus-org.asamk.Signal.service` are enabled.

I found [this](https://knx-user-forum.de/forum/supportforen/openhab/1139194-whisper-systems-signal-messenger-client-einrichten) on a German forum, and tried to understand it thanks to Google Translate. FlyingEaglE told in the comments that he got the same error message and could solve it by doing
`dpkg-reconfigure dbus`
This didn't work for me.

Any idea? I'm just beginning with dbus interfaces, I'm not a pro at all, so please detail your answer a bit.

Thanks,
Vic

**Comment by KRtekTM** [comment]
Hello, 

did you tried to view all DBus services through qdbusviewer or another program?

You should see org.asamk.Signal interface with all method published to DBus. If there some problem with DBus itself, qdbusviewer should fail or tell you something more about the issue.

**Comment by KRtekTM** [comment]
For example this is my system dbus shown by qdbusviewer (through putty on windows).

![2018-06-20 19_47_19-photos](https://user-images.githubusercontent.com/6234196/41675356-cfd83150-74c2-11e8-90c7-46cedd7bc68e.png)

**Comment by Vic3198** [comment]
Okay finally I found, it was so obvious that I didn't thought about it in the first time: it was a problem with `java.library.path`.
I don't know why I wanted to add `/usr/lib/jni` (where `libunix-java.so` is located) to my java.library.path. Finally I just copied `libunix-java.so` to a directory that already was in `java.library.path` and it worked! signal.service started without error and now I can send messages through DBus!
